### PR TITLE
LPS-180834 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LPS-180834

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:test:object

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectUpgrade.testcase [hunk 1]
```diff
@@ -13,7 +13,8 @@ definition {
 
 		AssertLocation(value1 = "${portalURL}/web/guest?SM_USER=test@liferay.com");
 
-		SearchAdministration.executeReindex();
+		//SearchAdministration.executeReindex();
+
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectUpgrade.testcase:17]: Why are we no longer reindexing here? We expect users to reindex after upgrades. Is it causing an issue with the update tests?

**Review by timpak — COMMENTED** [review]
@epark18 can you take a look at my question below?

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectUpgrade.testcase (hunk 1 above)
Comment (epark18) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectUpgrade.testcase:17]: Oops I commented that out while testing locally to save some time, and forgot to change it back.

**Comment by epark18** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:test:sf

**Review by timpak — APPROVED** [review]

**Comment by timpak** [comment]
ci:forward

**Pull request closed without merging**